### PR TITLE
Update for new API 2019-08-01 (#29)

### PR DESCRIPTION
**Pull request by leonardbf**
Includes new properties in volume response for upcoming replication functionality.
Change to tag type due to swagger review comment.
Inclusion of mount target type.
Internal ticket ANF-439

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.3'
+ModuleVersion = '0.1.4'
```
Comment (VeryEarly) [inline src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1]: no need to upgrade module version, will be updated during release

File: src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1 [hunk 2]
```diff
@@ -125,7 +125,7 @@ PrivateData = @{
 * Snapshot creation date now named just ''created'''
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```
Comment (VeryEarly) [inline src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1]: version 0.* is already considered preview, no need to add this field.

Comment (leonardbf) [reply src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1]: > version 0.* is already considered preview, no need to add this field.

Yes. However there was confusion on this point recently resulting in an email thread between NetApp and @yuemlu and @paulomarquesc among others. It was mentioned that this field should be added but also that the documentation stating that is now outdated. Given that there have also been review comments on previous PS updates rejecting the changes as breaking (despite the version number) I thought I would add in the text to make it extra clear

File: src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1 (hunk 1 above)
Comment (leonardbf) [reply src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1]: Ok, reverted.

**Pull request merged**